haskell/repositories.bzl

```diff
@@ -21,12 +16,20 @@ def rules_haskell_dependencies():
             urls = ["https://github.com/bazelbuild/bazel-skylib/archive/0.6.0.tar.gz"],
         )
 
+    if "rules_sh" not in excludes:
+        http_archive(
+            name = "rules_sh",
+            sha256 = "510d3428b7cdc85049cbd4ba73aeaa63b4aae579dd0a0a45537ddf40d807e281",
+            strip_prefix = "rules_sh-cd8cf4172618b0486c373afc7075256b3f8aa0aa",
+            urls = ["https://github.com/tweag/rules_sh/archive/cd8cf4172618b0486c373afc7075256b3f8aa0aa.tar.gz"],
```

Comment:
Would be good if we could rely on a release of `rules_sh` here. Not great to have `master` depend on unreleased versions of dependencies.